app/services/importers/create_call_off_contract.rb

```diff
@@ -79,17 +79,19 @@ def rows
     end
 
     def create_call_off_contract!(lead_provider:, contract_data:, cohort:)
-      CallOffContract.create!(
+      attributes = {
         lead_provider:,
         cohort:,
         uplift_target: contract_data[:uplift_target],
-        uplift_amount: contract_data[:uplift_amount],
         recruitment_target: contract_data[:recruitment_target],
         revised_target: contract_data[:revised_target],
         set_up_fee: contract_data[:set_up_fee],
         monthly_service_fee: contract_data[:monthly_service_fee],
         raw: contract_data.to_json,
-      )
+      }
+
+      attributes.merge!(uplift_amount: contract_data[:uplift_amount]) unless CallOffContract::COHORTS_WITH_NO_UPLIFT.include?(cohort.start_year)
```

Comment:
I'm not sure about coupling this to `COHORTS_WITH_NO_UPLIFT`
The CSV will set a nil value for `uplift_amount` and therefore will disable it.

Reply:
I think I agree, the `nil` value will be the flag instead of the global variable, let us know your thoughts :)

Reply:
Agree.. but I had to keep `COHORTS_WITH_NO_UPLIFT` in the importer context at least so we know to not include uplift fees for 2025 cohort when creating contracts seed data without a CSV.